### ISSUE TITLE
What's the difference between stats/cityscapes_mean.npy and  stats/cityscapes_mean_reported.npy?

### ISSUE THREAD

**Issue by gaozhitong** [shjung13/Standardized-max-logits#8]
Hi, as the title described, I notice that the code contains two mean/var statistics files in the stats directory. I am wondering what is the difference.

**Comment by farhadi76m** [comment]
It's also my question! 
How did he calculate the mean and var. was it for all images in train set or test ? it isn't clear to me !
